docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -838,7 +837,7 @@ When I select next value with `0.1` offset in picker wheel located `xpath(//XCUI
 === Recording iOS Performance metrics
 
 Records performance metrics of iOS application. It is based on the Instruments utility distributed by Xcode which provides a number of built-in analyses and measurements.
-See https://appiumpro.com/editions/12-capturing-performance-data-for-native-ios-apps[Capturing Performance Data for Native iOS Apps] and https://developer.apple.com/videos/play/wwdc2019/411/[Getting Started with Instruments] for more information.
+See https://appiumpro.com/editions/12-capturing-performance-data-for-native-ios-applications[Capturing Performance Data for Native iOS Apps] and https://developer.apple.com/videos/play/wwdc2019/411/[Getting Started with Instruments] for more information.
```

Comment:
"replace all" doesn't work well for all cases

Reply:
IDEA refactoring did this :O